tests/modules/login_user/app/login_user_usecase.test.ts

```diff
@@ -0,0 +1,47 @@
+import { it, expect, describe } from 'vitest'
+import { UserRepositoryMock } from '../../../../src/shared/infra/repositories/user_repository_mock'
+import { LoginUserUsecase } from '../../../../src/modules/login_user/app/login_user_usecase'
+import { ROLE } from '../../../../src/shared/domain/enums/role_enum'
+import { NoItemsFound } from '../../../../src/shared/helpers/errors/usecase_errors'
+
+describe('Assert Login User usecase is correct', () => {
```

Comment:
Same thing here, after adding the validation i said in the usecase make more test here